src/transformers/training_args.py

```diff
@@ -526,6 +526,10 @@ class TrainingArguments:
         default=False,
         metadata={"help": "Enable Ort"},
     )
+    ort_config_file: Optional[str] = field(
```

Comment:
Is it possible to merge both ort and ort_config_file into a single flag like deepspeed? --ort could optionally be followed by a json file and load_from_json will be called only when json file is specified.